logging/cluster-logging-deploying.adoc

```diff
@@ -24,8 +24,18 @@ The process for deploying cluster logging to {product-title} involves:
 
 include::modules/cluster-logging-deploy-console.adoc[leveloffset=+1]
 
+.Post install
+
+If you plan to use Kibana, after the Elasticsearch Operator and Cluster Logging Operator are fully installed, you must xref:#cluster-logging-visualizer-indices_cluster-logging-deploying[manually create your Kibana index patterns and visualizations] to explore and visualize data in Kibana.
```

Comment:
@bobfuru I am trying to say is that before you can use the Kibana dashboard to view logs, once Elasticsearch and Cluster Logging are installed and configured, you need to do these things. If you don't use Kibana, you don't need to. 
Does what I have make sense in that context? 
I think if I remove the comma, it sounds like you can use Kibana before Elasticsearch and Cluster Logging, which you cannot do.

Reply:
That pretty much makes sense, it's just hard to pick up on the order and steps with that clause in the middle. You could also try:

> If you plan to use Kibana to explore and visualize data, you must install the Elasticsearch Operator and Cluster Logging Operator, and  then you must manually create your Kibana index patterns and visualizations.

or 

> If you plan to use Kibana, you must manually create your Kibana index patterns and visualizations to explore and visualize data in Kibana after you have installed the Elasticsearch Operator and Cluster Logging Operator.

or bullets:

> If you plan to use Kibana to explore and visualize data, you must:
> 
> 1. Install the Elasticsearch Operator and Cluster Logging Operator.
> 2. Manually create your Kibana index patterns and visualizations.

Reply:
@bobfuru Or, can we remove the _ after the Elasticsearch Operator and Cluster Logging Operator are fully installed,_ where it is a post-installation task. Isn't that assumed?

_If you plan to use Kibana, you must manually create your Kibana index patterns and visualizations to explore and visualize data in Kibana._

Reply:
Can I also ask if the note I added here [1] is confusing? The issue is, we added a note that using the Log Forward API is optional. The default behavior of cluster logging is to send most logs to the internal Elasticsearch log store. We added a note to this effect, as some folks thought they now needed to use the API to forward logs to the internal ES log store.

However, due to lack of security in the log store, audit logs are not sent by default. I added a note stating that the user needs to use the Log Forward API if they want the audit logs.

[1] https://github.com/openshift/openshift-docs/pull/24579/commits/bfa73083d23c69e9e7d7ee0cbc2a32a42e80ddd2

Reply:
For the first question, it does make sense to me to remove the part about Operator install. I wasn't sure if you had that in there before to avoid a common gotcha. If not, then it's easier to follow without. 👍

Reply:
For the second question about the Log Forward API note, it makes sense to me. To me, it's logical that they would want to know that it's optional but necessary if they want to send logs to the internal store.